src/ui/buttons/buttonDesigns/index.js

```diff
@@ -31,6 +37,12 @@ const DESIGN_MAP : Object = {
         getValidDesignProps:   getValidInlineLogoTextProps,
         designConfig:          INLINE_LOGO_TEXT_CONFIG,
         ButtonDesignComponent: InlineLogoTextComponent
+    },
+    'run-fadeout-logo-show-label-text-design': {
```

Comment:
This file looks good. In xobuyer the id for this design is `alternate-slide-logo-animation` so could you use that name in your changes?

Reply:
I can change this animation id, but in my humble opinion, alternate-slide-logo-animation is not clear enough to understand what the animation is about, so I will prefer to use fadeoutLogoShowLabelText in my functions and file